k8s/namespaces/admin/aad-pod-identity/patches/sbox/aad-pod-id.yaml

```diff
@@ -15,6 +15,7 @@ spec:
         key: CriticalAddonsOnly
         operator: Equal
         value: "true"
+      nodeSelector:  # This can be removed once the changes to use the system node pool are in prod
```

Comment:
why does a daemonset need a node selector?

Reply:
We don't need it as we're using taints, but it's in the config https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.8.4-rc.0/deploy/infra/deployment-rbac.yaml

```
      nodeSelector:
        kubernetes.io/os: linux
```

Reply:
There's charts available, maybe we should try using those in the future

Reply:
hold on, the node selector is to prevent it being scheduled on windows because the container doesn't work on windows, why are you removing this?